src/System.Security.Cryptography.Xml/ref/Configurations.props

```diff
@@ -2,8 +2,8 @@
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
     <BuildConfigurations>
-      netcoreapp;
-      uap;
+      netstandard;
+      uap
```

Comment:
I don't know how to resolve that. @weshaggard 

I guess I want to understand why it's needed.